src/main/java/de/dhbw/karlsruhe/top/down/parsing/models/TopDownState.java

```diff
@@ -0,0 +1,7 @@
+package de.dhbw.karlsruhe.top.down.parsing.models;
+
+public enum TopDownState {
+    Z0,
+    Z,
+    ZF
+}
```

Comment:
Vielleicht den BottomUpState und den TopDownState zusammenführen als ParsingState oder so und dann natürlich in allen Beispiel-Parsern fürs Testen anpassen.

Reply:
Du hast einen ParsersState hinzugefügt , jetzt kann die Klasse auch entfernt werden :)